app/views/ChargeSummary.scala.html

```diff
@@ -181,6 +189,8 @@
     case (PaymentOnAccountTwoReviewAndReconcile, _)                                              => getPoa2ExtraChargeInterestExplanationContent
     case (PaymentOnAccountOne | PaymentOnAccountTwo, _) if !viewModel.latePaymentInterestCharge  => paymentOnAccountExplanation
     case (PaymentOnAccountOne | PaymentOnAccountTwo, _)                                          => getLpiPoaExplanationContent
+    case (PaymentOnAccountOneReviewAndReconcileCredit, _)                                        => reviewAndReconcilePoaCreditExplanation("1")
```

Comment:
as per comment above, please make passing params values more readable

Reply:
I think I'm going to make a tech debt ticket to shorten a lot of these names, so to change maybe

PaymentOnAccountOne -> PoaOneCharge
PaymentOnAccountOneReviewAndReconcile -> PoaOneReconcileCharge
PaymentOnAccountOneReviewAndReconcileCredit -> PoaOneReconcileCredit
etc

Reply:
[MISUV-8655](https://jira.tools.tax.service.gov.uk/browse/MISUV-8655)